pkg/daemon/certs/certs.go

```diff
@@ -0,0 +1,125 @@
+package certs
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"syscall"
+	"unsafe"
+
+	syswindows "golang.org/x/sys/windows"
+	"k8s.io/klog/v2"
+)
+
+// Reconcile ensures the certificates in the given trusted CA bundle file are imported as system certificates.
+// Returns a boolean whether any certs have needed to be imported
+func Reconcile(trustBundle string) (bool, error) {
+	if trustBundle == "" {
+		// TODO: ensure that all certs that we've added previously are removed https://issues.redhat.com/browse/WINC-998
+		return false, nil
+	}
+	// Read expected certs from CA trust bundle file
+	pemData, err := ioutil.ReadFile(trustBundle)
```

Comment:
Isn't `ioutil.ReadFile` deprecated? In general we shouldn't we trying to read the whole file in one shot. There could be potential of a DOS. It might be better to use `os.Open()` and then `io.Copy()`. Using a context aware reader will also help.

Reply:
Yup, it's deprecated, I'll use os.Open() and then io.Copy() instead.

I also can explore reading in the file one cert at a time, maybe using the `-----END CERTIFICATE-----` as a delimiter to the buffered reader